SPECS/php/plans/all.fmf

```diff
@@ -0,0 +1,6 @@
+summary: Test plan with all beakerlib tests
```

Comment:
What is this file and why is it here? We usually don't do subdirectories for spec sources.

Reply:
This folder got copied over by accident from the Fedora spec. Removing it.